### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by DominiqueMakowski**

### CODE REVIEW

File: R/convert_tFz_to_r.R [hunk 1]
```diff
@@ -91,12 +91,35 @@ r_to_t <- function(r, n = NULL, df_error = NULL, ...){
...
+#' @rdname t_to_r
+#' @export
+r_to_z <- function(r, n = NULL, df_error = NULL, ...) {
+  if(is.null(n) & !is.null(df_error)){
+    n <- df_error + 2
+  }
+  if(is.null(n)){
+    # Zar, J.H., (2014). Spearman Rank Correlation: Overview. Wiley StatsRef: Statistics Reference Online. doi:10.1002/9781118445112.stat05964
+    # TODO: add variants for Spearman and Kendall (Zar, 2014)
+    atanh(r)  # Fisher z-transformation
```
Comment (mattansb) [inline R/convert_tFz_to_r.R]: @DominiqueMakowski 
Fisher's transformation z is not the same as the test statistic z - the first is more of "z score" on the parameter scale, whereas the second is on the sampling distribution scale.
Essentially, to get from one to the other you need N:

``` r
z <- 4
n <- 100

effectsize::z_to_r(z, n = n)
#> [1] 0.3713907

# first transform stat-z to Fisher-z, only then take the tan:
fishers_z <- z / sqrt(n - 2)
tanh(fishers_z)
#> [1] 0.3834184
```

<sup>Created on 2019-11-17 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

And so, if you need n anyway, and both methods give similar approximations of one another, is there a need for this method?

You could have a separate function for Fisher's transformations, but that would be:
```
Fishers_z_to_r <- tann
r_to_Fishers_z <- atann
```

which is maybe a bit silly... ^_^


Note also that 2 * fishers z is basically what we're going for in `z_to_d`:
``` r
z <- -2.5
n <- 111

fishers_z <- z / sqrt(n - 2)

effectsize::z_to_d(z, n = n)
#> [1] -0.474579
2 * fishers_z
#> [1] -0.4789131
```

<sup>Created on 2019-11-17 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Comment (DominiqueMakowski) [reply R/convert_tFz_to_r.R]: I need Fisher's z for the correlation package (to get CIs)

I was too quick on assuming the equivalence between Fisher'z and "regular" z..., in that case, lI suggest including a separate Fisher's z function in correlation (`z_fisher()`) where it'll be primarily used.

Comment (mattansb) [reply R/convert_tFz_to_r.R]: Fisher was not very creative with the letters he assigned to his formulas... Can we email him to complain?

Thanks for tagging me!

Comment (DominiqueMakowski) [reply R/convert_tFz_to_r.R]: you mean Fisher is a person and do not refer to the "Fishermen and statisticians" club? 🐟 

Thanks for correcting ;)

**Comment by strengejacke** [comment]
> This branch has conflicts that must be resolved

😒 

wasn't me! ... but, I don't know how to solve conflicts from the command line.

**Pull request merged**